packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 46.7.0 - July 25, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-visualforce-language-server
+
+- Update package to depend on latest version of vscode-\* modules ([PR #1486](https://github.com/forcedotcom/salesforcedx-vscode/pull/1486))
```

Comment:
I think this change can be worded with something like "Update to the latest language server version"